cmd/kiya/flags.go

```diff
@@ -11,4 +11,9 @@ var (
 	oOutputFilename = flag.String("o", "", "if not empty then write the secret to a file else write to stdout (get)")
 	oCreatedAt      = flag.String("ca", "", "if not empty then use this formatted timestamp (RFC3339) as the creation date of the new secret")
 	oQuiet          = flag.Bool("quiet", false, "don't prompt for confirmation on destructive actions")
+
+	oEncrypted   = flag.Bool("encrypted", false, "encryptFile backups")
```

Comment:
I feel like I'm missing something here in the description? Also minor for the line below where it ends with a space.